### PR TITLE
Issue#666

### PR DESCRIPTION
**Pull request by lazarm**
Twitter login

### CODE REVIEW

**Comment by lazarm** [comment]
Please don't say i have to cherry-pick commits :-)

**Comment by MartinJakomin** [comment]
I think I can answer in Mitar's name:
Do the Cherry-pick :)

File: piplmesh/account/backends.py [hunk 1]
```diff
@@ -76,3 +76,23 @@ def authenticate(self, token=None, request=None):
         user.save()
 
         return user
+
+
+
+import tweepy
```
Comment (MartinJakomin) [inline piplmesh/account/backends.py:82]: imports should be at the top

**Comment by mitar** [comment]
I think Martin said it all ... ;-)

**Comment by lazarm** [comment]
okay, creating new issue

**Pull request closed without merging**